adapters/klab.ogc/src/main/java/org/integratedmodelling/klab/stac/STACAssetMapParser.java

```diff
@@ -0,0 +1,15 @@
+package org.integratedmodelling.klab.stac;
+
+import java.util.Set;
+
+import kong.unirest.json.JSONObject;
```

Comment:
In the project there seems to be different classes for handling `JSONObject`.
* `kong.unirest.json.JSONObject`
* `com.nimbusds.jose.shaded.json.JSONObject`
* `net.minidev.json.JSONObject`
* `org.json.JSONObject`

Any idea why this one is preferred over the other ones? 🤔

Reply:
I am obtaining those JSONs via HTTP calls done with `kong.Unirest`, so I kept using that library. I do not know which one would fit our needs better.

This is not the only context in which we have a multiple libraries that do basically the same thing. We should do something about that.